kythe/go/platform/tools/kindex_file_generator/BUILD

```diff
@@ -26,7 +26,7 @@ shell_tool_test(
       --output "$$kindex_output" \\
       --source "$$input"
 
-    size="$$(du -b "$$kindex_output" | cut -f1)"
+    size="$$(wc -c < "$$kindex_output")"
     if [[ "$$size" -eq 0 ]] ; then
```

Comment:
I think this could also be more simply

    if [[ -s "$$kindex_output" ]] ; then

Reply:
Er, sorry, that's `! -s "$$kindex_output"`